README.md

```diff
@@ -10,18 +10,16 @@ This package enriches your Fivetran data by doing the following:
 
 ## Models
 
-This package contains staging models, designed to work simultaneously with our [Bing Ads modeling package](https://github.com/fivetran/dbt_bing_ads). The staging models:
-
-* Name columns consistently across all packages:
-* Boolean fields are prefixed with is_ or has_
-* Timestamps are appended with _at
-* ID primary keys are prefixed with the name of the table. For example, the campaign table's ID column is renamed campaign_id.
+This package contains staging models, designed to work simultaneously with our [Microsoft Advertising modeling package](https://github.com/fivetran/dbt_microsoft_ads). The staging models name columns consistently across all packages:
+* Boolean fields are prefixed with `is_` or `has_`
+* Timestamps are appended with `_at`
+* ID primary keys are prefixed with the name of the table. For example, the campaign table's ID column is renamed `campaign_id`.
```

Comment:
I removed the "name columns" bullet point. We should only use bulleted lists if we have two or more items.